core/commands/ping.go

```diff
@@ -41,7 +41,7 @@ trip latency information.
 		cmds.StringArg("peer ID", true, true, "ID of peer to be pinged.").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.IntOption("count", "n", "Number of ping messages to send."),
+		cmds.IntOption("count", "n", "Number of ping messages to send.").Default(10),
```

Comment:
Oh! Default actually uses the value passed, and doesn't just format it as text? Ok, that makes things more complex. I'll need to check this over, then.